src/main/java/io/strimzi/kafka/bridge/mqtt/core/MqttServerHandler.java

```diff
@@ -85,6 +85,13 @@ public void channelActive(ChannelHandlerContext ctx) {
 
     @Override
     protected void channelRead0(ChannelHandlerContext ctx, MqttMessage msg) {
+        if (msg.decoderResult() != null) {
+            Throwable error = msg.decoderResult().cause();
+            if (error != null) {
+                exceptionCaught(ctx, error);
```

Comment:
I think it would be useful to have an explicit error logged here, as well as printing the stack trace. Something like `LOGGER.error("Received invalid message with error: {}, cause.getMessage())`